src/main/webapp/script.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Fetches comments from the server and adds them to the DOM. */
+function loadComments() {
+  fetch('/display-comments').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
+/** Redirects the user to the result page, with the given parameter for user-query. */
+function redirectToResults(userQuery) {
+  document.location.href = "search-results-test.html?user-query=" + userQuery;
+  return false;
```

Comment:
What is this return for?

Reply:
WOW I DID NOT SEE THESE COMMENTS FOR SOME REASON SORRY
the return is because the page reloads without the return and then it yeets the search results LOL